pkg/api/event/nonblocking/identity_verified.go

```diff
@@ -25,16 +25,20 @@ func NewIdentityVerifiedEventPayload(
 	claimName string,
 	adminAPI bool,
 ) *IdentityVerifiedEventPayload {
```

Comment:
It is better to change the return type to  `(*IdentityVerifiedEventPayload, bool)` if the payload can be nil sometimes.